bundles/admin/admin-layereditor/view/AdminLayerForm/PermissionTabPane/PermissionRow.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const RowContainer = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+`;
+
+const TextColumn = styled.div`
+    flex-grow: 1;
+    width: 195px;
+    padding-left: 5px;
+    align-self: ${props => props.isHeaderRow ? 'flex-end' : 'stretch'};
+`;
+
+const StyledPermissionDiv = styled.div`
+    flex-grow: 1;
+    float: left;
+    width: 110px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 5px !important;
+`;
+
+const Break = styled.div`
+    flex-basis: 100%;
+    height: 0;
+`;
+
+const HeaderPermissionText = styled.div`
+    height: 80%;
+    text-align: center;
+    padding-bottom: 5px;
+`;
+
+const SelectAllDiv = styled.div`
+    height: 20%;
+    padding-bottom: 5px;
+`;
+
+export const PermissionRow = (props) => {
+    const checkboxDivs = [];
+    props.checkboxes.forEach(checkbox => {
+        if (props.isHeaderRow) {
+            checkboxDivs.push(<StyledPermissionDiv key={checkbox.key}><HeaderPermissionText >{checkbox.props.selectionText}</HeaderPermissionText><Break/><SelectAllDiv>{checkbox}</SelectAllDiv></StyledPermissionDiv>);
+        } else {
+            checkboxDivs.push(<StyledPermissionDiv key={checkbox.key}>{checkbox}</StyledPermissionDiv>);
+        }
+    });
```

Comment:
Would this work/be more readable:
```
    const checkboxDivs = props.checkboxes.map(checkbox => {
        const content = checkbox;
        if (props.isHeaderRow) {
            content = (<HeaderPermissionText >{checkbox.props.selectionText}</HeaderPermissionText><Break/><SelectAllDiv>{content}</SelectAllDiv>);
        } 
        return (<StyledPermissionDiv key={checkbox.key}>{content}</StyledPermissionDiv>);
    )};
```
Or maybe even:
```
    const checkboxDivs = props.checkboxes.map(checkbox => {
        const content = checkbox;
        if (props.isHeaderRow) {
            content = (
                <React.Fragment>
                    <HeaderPermissionText>{checkbox.props.selectionText}</HeaderPermissionText>
                    <Break/>
                    <SelectAllDiv>{content}</SelectAllDiv>
                </React.Fragment>
            );
        } 
        return (
            <StyledPermissionDiv key={checkbox.key}>
                {content}
            </StyledPermissionDiv>
        );
    )};

Reply:
Fixed as suggested